app/src/main/java/de/danoeh/antennapod/adapter/AllEpisodesListAdapter.java

```diff
@@ -125,6 +128,22 @@ public View getView(int position, View convertView, ViewGroup parent) {
                 holder.progress.setVisibility(View.GONE);
             }
 
+            TypedArray typeDrawables = context.obtainStyledAttributes(
+                    new int[]{R.attr.type_audio, R.attr.type_video});
+            final int[] labels = new int[]{R.string.media_type_audio_label, R.string.media_type_video_label};
+            MediaType mediaType = item.getMedia().getMediaType();
```

Comment:
I should perhaps have mentioned it: that code (starting with ```TypedArray typeDrawables = context.obtainStyledAttributes``` was copied from FeedItemlistAdapter.java. There you also have `media` already defined before the enclosing if-statement.